vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datepicker/DatePickerLocaleIT.java

```diff
@@ -186,6 +187,32 @@ public void datePickerWithLocale_setInputValue_blur_assertDisplayedValue() {
         assertNoWarnings();
     }
 
+    @Test
+    public void testLocaleBasedParsingShouldUseDefaultReferenceDateIfNotSpecified() {
```

Comment:
Test naming convention is different in this file, would be good to keep new tests consistent with the existing naming scheme.